internal/website/themes/hyde/layouts/partials/head.html

```diff
@@ -23,7 +23,7 @@
 
   <!-- Icons -->
   <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
```

Comment:
Optional: This icon probably does not match with your Go icon. Probably best to remove it for the time being.

Reply:
Good idea, will do